packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/create/iti39-envelope.ts

```diff
@@ -111,8 +112,19 @@ export function createAndSignBulkDRRequests({
   const signedRequests: BulkSignedDR[] = [];
 
   for (const bodyData of bulkBodyData) {
-    const signedRequest = createAndSignDRRequest(bodyData, samlCertsAndKeys);
-    signedRequests.push({ gateway: bodyData.gateway, signedRequest, outboundRequest: bodyData });
+    const documentReferences = bodyData.documentReference;
+    for (let i = 0; i < documentReferences.length; i += maxDocumentReferencesPerDrRequest) {
```

Comment:
many epic endpoints fail if we ask for more than 10 documents. This happens in prod often. Batching DR requests into 10 document refrerence chunks

Reply:
redox endpoints are failing if we send them more than one DR at a time